Source/Flow/Private/FlowComponent.cpp

```diff
@@ -44,6 +44,11 @@ void UFlowComponent::BeginPlay()
 {
 	Super::BeginPlay();
 
+	RegisterWithFlowSubsystem();
```

Comment:
Our object pooling model required different timing for registration with the subsystem than strictly in Begin/End Play, so we split these parts out into dedicated functions